### ISSUE TITLE
Model training accuracy issue

### ISSUE THREAD

**Issue by kamen007** [YuYue525/MobileSAM-pytorch#7]
May I know what is the MSE after your training? My MSE is 0.6, which feels a bit high.
Thanks for your great work.

**Comment by YuYue525** [comment]
Cannot remember the exact number but very low, even lower than the model weight provided by official MobileSAM. 0.6 is too high, it must be something wrong.

**Comment by andyqmongo** [comment]
The MSE loss of my re-trained MobileSAM is also 0.6, the original MobileSAM MSE is around 1~2 during evaluation.
I use the defualt hyperparamter for training, except changing the batch size to 12.
![image](https://github.com/YuYue525/MobileSAM-pytorch/assets/20841667/30a6710b-5ded-4585-a4fe-401731284d4e)
![image](https://github.com/YuYue525/MobileSAM-pytorch/assets/20841667/bd2741f6-f70f-4d29-ac59-6ca9a9bcedeb)

**Comment by YuYue525** [comment]
Great! Apologize for the mistake in my previous answer because I really don’t remember the exact value of loss.

**Comment by kamen007** [comment]
> The MSE loss of my re-trained MobileSAM is also 0.6, the original MobileSAM MSE is around 1~2 during evaluation. I use the defualt hyperparamter for training, except changing the batch size to 12. ![image](https://user-images.githubusercontent.com/20841667/284411869-30a6710b-5ded-4585-a4fe-401731284d4e.png) ![image](https://user-images.githubusercontent.com/20841667/284411320-bd2741f6-f70f-4d29-ac59-6ca9a9bcedeb.png)
> 
> The performance seems on par with the original MobileSAM. (currently, eval on 1300 images with `eval_seg_everything.py`) Original: mAP: 0.642 mIoU: 0.701 mSSIM: 0.756 re-trained: mAP: 0.676 mIoU: 0.661 mSSIM: 0.753

my experimental results are similar to yours.

**Comment by SunIsland-L** [comment]
Hello, I see that the 2%SA-1B dataset is used in the code for training. Will the effect be worse if it is aligned with the 1%SA-1B in the MobileSAM paper?

**Comment by Vickeyhw** [comment]
@andyqmongo  Hi~ How many epochs and data you used to produce the results? It would be better if you could provide some other hyperparameters such as learning rate\batch size. Thank you very much!
